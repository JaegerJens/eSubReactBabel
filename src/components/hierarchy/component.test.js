import React from 'React';
import Hierarchy from './component.jsx';
import renderer from 'react-test-renderer';

describe('Hierarchy component', () => {
    it('matches snapshot', () => {
        const prop = {
            data: ['a', 'b', 'c'],
            selected: 2,
            onSelect: "onSelect function"
        };
        expect(Hierarchy(prop)).toMatchSnapshot();
    });
});