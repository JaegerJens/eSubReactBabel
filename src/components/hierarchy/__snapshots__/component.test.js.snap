// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hierarchy component matches snapshot 1`] = `
<div>
  <h2>
    Hierarchy View
  </h2>
  <Tree
    data={
      Array [
        "a",
        "b",
        "c",
      ]
    }
    onClick="onSelect function"
    select={2}
  />
</div>
`;
